browser/ui/webui/ai_chat/ai_chat_ui_page_handler.h

```diff
@@ -36,13 +33,13 @@ class FaviconService;
 
 namespace ai_chat {
 class AIChatUIPageHandler : public ai_chat::mojom::PageHandler,
-                            public TabStripModelObserver,
                             public AIChatTabHelper::Observer,
                             public content::WebContentsObserver {
  public:
   AIChatUIPageHandler(
       content::WebContents* owner_web_contents,
-      TabStripModel* tab_strip_model,
+      bool is_standalone,
```

Comment:
I wonder if we even need `is_standalone`? Perhaps consumer just passes `nullptr` for the `WebContents` param?

Reply:
yeah it could be detected that way, I will change it